js/VideoView.js

```diff
@@ -30,6 +32,7 @@ export default class VideoView extends Backbone.View {
   setUpListeners() {
     this.$el.on('onscreen', this.onScreenChange);
     this.listenTo(Adapt, 'device:resize', this.render);
+    documentModifications.on('changed:html', this.checkVisua11y);
```

Comment:
Sorry, I just remembered that `documentModifications` is a Backbone.Events interface.

```suggestion
    this.listenTo(documentModifications, 'changed:html', this.checkVisua11y);
```